src/test/java/com/jwplayer/southpaw/state/RocksDBStateAutoRollbackTest.java

```diff
@@ -1,178 +0,0 @@
-/*
```

Comment:
Note these tests were merged into `src/test/java/com/jwplayer/southpaw/state/RocksDBStateTest.java` due to the test setup being refactored to allow different RocksDB configuration options to be set